sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-backend-v2/README.md

```diff
@@ -80,10 +80,10 @@ spring.security.oauth2.client.registration.azure.redirect-uri-template={baseUrl}
 spring.security.oauth2.client.registration.azure.client-authentication-method=basic
 spring.security.oauth2.client.registration.azure.authorization-grant-type=authorization_code
 
-spring.security.oauth2.client.provider.azure-oauth-provider.authorization-uri=https://login.microsoftonline.com/common/oauth2/authorize
-spring.security.oauth2.client.provider.azure-oauth-provider.token-uri=https://login.microsoftonline.com/common/oauth2/token
-spring.security.oauth2.client.provider.azure-oauth-provider.user-info-uri=https://login.microsoftonline.com/common/openid/userinfo
-spring.security.oauth2.client.provider.azure-oauth-provider.jwk-set-uri=https://login.microsoftonline.com/common/discovery/keys
+spring.security.oauth2.client.provider.azure.authorization-uri=https://login.microsoftonline.com/common/oauth2/v2.0/authorize
```

Comment:
Why update `azure-oauth-provider` to `azure`? the provider name is `azure-oauth-provider` here, see L77